test/webhook_test.js

```diff
@@ -1,18 +1,22 @@
-suite('webhook', () => {
-  let helper = require('./helper');
-  let assert = require('assert');
+const helper = require('./helper');
+const assert = require('assert');
+
+helper.secrets.mockSuite('webhook', ['taskcluster'], function(mock, skipping) {
+  helper.withEntities(mock, skipping);
+  helper.withFakeGithub(mock, skipping);
+  helper.withServer(mock, skipping);
 
   let github = null;
 
-  setup(async () => {
+  setup(async function() {
```

Comment:
why ditch arrow syntax?

Reply:
The mocha functions all pass a context that is useful for things like `this.pending = true`, and to get a context you need to use `function` and not an arrow.  Most of the time it doesn't matter, but it's easier to just be consistent about it.

Reply:
wow, I didn't know this! ok  👍